src/core/server/http/lifecycle/on_pre_routing.ts

```diff
@@ -102,24 +102,7 @@ export function adoptToHapiOnRequest(fn: OnPreRoutingHandler, log: Logger) {
         appState.rewrittenUrl = appState.rewrittenUrl ?? request.url;
 
         const { url } = result;
-
-        // TODO: Remove once we upgrade to Node.js 12!
```

Comment:
added by @watson to fix a CI failure https://github.com/elastic/kibana/pull/80713